src/isar_exr/api/energy_robotics_api.py

```diff
@@ -385,7 +437,8 @@ def wake_up_robot(
             ).total_seconds()
             if time_passed_since_function_call > timeout:
                 raise RobotException(
```

Comment:
Why not raise `RobotMissionStatusException` and just pass the error description?